lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.pm

```diff
@@ -55,19 +62,31 @@ sub _copy_config_profile_items_to_project {
     my @config_profile_items = @_;
 
     for my $original_config_item (@config_profile_items) {
+        my $new_item;
         if ($original_config_item->analysis_menu_item) {
-            Genome::Config::Profile::Item->create(
+            $new_item = Genome::Config::Profile::Item->create(
                 analysis_project => $analysis_project,
                 analysis_menu_item => $original_config_item->analysis_menu_item,
                 status => $original_config_item->status,
```

Comment:
I'm curious if you could get an array of tags and just pass `tags => \@tags` into the create calls?  Seems like `Genome::Config::Profile::Item` [tags][1] is setup for that.

[1]: https://github.com/tmooney/genome/blob/7cfc7a28d99558f0c2457562effc7283317e3965/lib/perl/Genome/Config/Profile/Item.pm#L54